src/Symfony/Component/Mailer/Bridge/Amazon/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ CHANGELOG
    to `Symfony\Component\Mailer\Bridge\Amazon\Transpor\SesApiTransport`, `Symfony\Component\Mailer\Bridge\Amazon\Http\SesTransport`
    to `Symfony\Component\Mailer\Bridge\Amazon\Transport\SesHttpTransport`, `Symfony\Component\Mailer\Bridge\Amazon\Smtp\SesTransport`
    to `Symfony\Component\Mailer\Bridge\Amazon\Transport\SesSmtpTransport`.
+ * [BC BREAK] changed `Symfony\Component\Mailer\Bridge\Amazon\Transpor\SesApiTransport::__construct` username and password arguments to credential
```

Comment:
Typo in namespace, `s/Transpor/Transport/`

Reply:
Oh copy paste from line 8. :)